x/emissions/keeper/queryserver/query_server_stake.go

```diff
@@ -34,6 +35,34 @@ func (qs queryServer) GetReputerStakeInTopic(ctx context.Context, req *types.Que
 	return &types.QueryReputerStakeInTopicResponse{Amount: stake}, nil
 }
 
+// Retrieves all stake in a topic for a given set of reputer addresses,
+// including their stake in themselves and stake delegated to them.
+// Also includes stake that is queued for removal.
+func (qs queryServer) GetMultiReputerStakeInTopic(ctx context.Context, req *types.QueryMultiReputerStakeInTopicRequest) (*types.QueryMultiReputerStakeInTopicResponse, error) {
+	maxLimit, err := qs.k.GetParamsMaxLimit(ctx)
+	if err != nil {
+		maxLimit = types.DefaultParamsMaxLimit()
+	}
+
+	if uint64(len(req.Addresses)) > maxLimit {
+		return nil, status.Error(codes.InvalidArgument, "cannot query more than 100 addresses at once")
```

Comment:
maxLimit is dynamic in the sense that it's set by params yeah? then this error should be 
```golang
status.Error(codes.InvalidArgument , fmt.sprintf("cannot query more than %d addresses at once", maxLimit))
```

Reply:
yessir